src/components/StudyList/StudyList.tsx

```diff
@@ -37,7 +32,7 @@ const StudyList: React.FC = () => {
     <>
       <IonListHeader>
         <IonLabel>Studies</IonLabel>
-        <IonButton routerLink={PATHS.STUDY_CREATE_PAGE}>New</IonButton>
+        <IonButton routerLink={paths.studyCreate}>New</IonButton>
```

Comment:
Since we also have "Edit" on the screen, I'm wondering whether using a verb here will result in people finding the UI more cohesive. The verb that first came to mind is "Create", but that seems long to me. The next one that came to mind is "Add". "New" definitely works for me for now—I don't want to hold up the merge for this.

Reply:
Okay, maybe that's something we can get input from the wider group on.